OloEditor/SandboxProject/Assets/Scripts/LuaScripts/LuaCamera.lua

```
@@ -0,0 +1,45 @@
+-- LuaCamera.lua — Lua port of Camera.cs
+-- A simple camera that follows the player entity with an offset.
+-- Attach via LuaScriptComponent with ScriptFile = "Scripts/LuaScripts/LuaCamera.lua"
+
+local LuaCamera = {}
+
+local distanceFromPlayer = 5.0
```

Luacheck (1.2.0)

[warning] unused variable 'distanceFromPlayer'

(W211)